tests/graph/test_contract_nodes.py

```diff
@@ -15,6 +15,27 @@
 import retworkx
 
 
+class UE(tuple):
```

Comment:
I'd probably name this `UndirectedEdge` just for clarity. When I saw `UE` at first I wasn't sure what that meant.

Reply:
Done in 44e0b79!